core/src/main/java/com/keenant/tabbed/util/Packets.java

```diff
@@ -7,49 +7,46 @@
 import com.comphenix.protocol.wrappers.PlayerInfoData;
 import org.bukkit.entity.Player;
 
-import java.lang.reflect.InvocationTargetException;
 import java.util.Collections;
 import java.util.List;
 
 /**
  * Some generic-ish packet utils.
  */
-public class Packets {
+public final class Packets {
+
     /**
      * Creates a PLAYER_INFO packet from the params.
-     * @param action
-     * @param data
-     * @return
      */
     public static PacketContainer getPacket(PlayerInfoAction action, PlayerInfoData data) {
         return getPacket(action, Collections.singletonList(data));
     }
+
     /**
      * Creates a PLAYER_INFO packet from the params.
-     * @param action
-     * @param data
-     * @return
      */
-
     public static PacketContainer getPacket(PlayerInfoAction action, List<PlayerInfoData> data) {
         PacketContainer packet = ProtocolLibrary.getProtocolManager().createPacket(Server.PLAYER_INFO);
         packet.getPlayerInfoAction().write(0, action);
         packet.getPlayerInfoDataLists().write(0, data);
         return packet;
     }
 
+    /**
+     * Creates a PLAYER_INFO_REMOVE packet.
+     */
+    public static PacketContainer getRemovePacket(List<PlayerInfoData> removedPlayers) {
+        PacketContainer packet = ProtocolLibrary.getProtocolManager().createPacket(Server.PLAYER_INFO_REMOVE);
+        packet.getPlayerInfoDataLists().write(0, removedPlayers);
+        return packet;
+    }
+
     /**
      * Sends a list of ProtocolLib packets to a player.
-     * @param player
-     * @param packets
-     * @return
      */
     public static void send(Player player, List<PacketContainer> packets) {
-        try {
-            for (PacketContainer packet : packets)
-                ProtocolLibrary.getProtocolManager().sendServerPacket(player, packet, false);
-        } catch (InvocationTargetException e) {
-            e.printStackTrace();
-        }
+        for (PacketContainer packet : packets)
```

Comment:
Revert removing of the try...catch
We don't want it to explode for everyone because of one slot for one user causes problems